aether-kernel/aether/kernel/api/utils.py

```diff
@@ -52,3 +56,125 @@ def merge_objects(source, target, direction):
     else:
         result = target
     return result
+
+
+def get_unique_schemas_used(mappings_ids):
+    result = {}
+    schemas = models.Schema.objects.filter(schemadecorators__mappings__id__in=mappings_ids)
+    for schema in schemas:
+        other_linked_mappings = models.Mapping.objects.filter(
+            schemadecorators__schema__id=schema.id
+        ).exclude(id__in=mappings_ids)
+        result[schema.name] = {
+            'id': schema.id,
+            'name': schema.definition['name'],
+        }
+        if other_linked_mappings:
```

Comment:
This condition can be changed to `result[schema.name]['is_unique'] = not other_linked_mappings` :nerd_face: